common/src/main/java/com/wynntils/features/inventory/ContainerSearchFeature.java

```diff
@@ -87,7 +89,7 @@ public void onScreenInit(ScreenInitEvent event) {
 
         currentSearchableContainerType = searchableContainerType;
 
-        addSearchWidget(screen, renderX, renderY);
+        addSearchWidget(((AbstractContainerScreen<ChestMenu>) screen), renderX, renderY);
```

Comment:
Can you replace ? with ChestMenu in line 78, so you can skip this cast? (Maybe that is impossible, the type erasure is hard to understand, but I think it should work)

Reply:
'Screen' cannot be safely cast to 'AbstractContainerScreen<ChestMenu>'